packages/react/.storybook/main.ts

```diff
@@ -19,16 +17,12 @@ const config: StorybookConfig = {
   async viteFinal(inputConfig, { configType }) {
     let config = inputConfig;
 
-    if (configType === 'PRODUCTION') {
```

Comment:
No longer need this since turbosnap is included by default now, apparently since Storybook 8:

<img width="912" height="744" alt="Image" src="https://github.com/user-attachments/assets/1dfd6692-dca5-473d-932c-e77b87e928eb" />